CHANGELOG.md

```diff
@@ -52,8 +52,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   used to generate a new change-id, which may help resolve some divergences.
 
 * `jj op show` now accepts `-T`/`--template` option to customize the operation
-  output using template expressions, similar to `jj op log`. Also added
-  `--no-op-diff` flag to suppress the operation diff.
+  output using template expressions, similar to `jj op log`. The default template
+  can be configured with `templates.op_show`. Also added `--no-op-diff` flag to
```

Comment:
nit: `templates.op_show` is a breaking change.

Reply:
I think it's breaking change because the user who customizes `op_log` will have to set `op_show`.